### PR TITLE
refactor: redis and middleware removed

### PR DESCRIPTION
**Pull request by abrehmaaan**

### CODE REVIEW

**Review by Fahad-Mahmood — COMMENTED** [review]
looks good, but please delete the code and files instead of commenting.

File: controllers/dagora.controllers.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-const { redisClient } = require('../lib/redisClient');
-const { generateAgoraThreadsRedisKey } = require('../util/generateRedisKeys');
+// const { redisClient } = require('../lib/redisClient');
+// const { generateAgoraThreadsRedisKey } = require('../util/generateRedisKeys');
 const { queryClientForThreadsByCategory, queryClientForRepliesByThreadId, queryClientForThreadById} = require('../helpers/queries');
-const { getCacheItemIndexFromList } = require('../helpers/cache');
+// const { getCacheItemIndexFromList } = require('../helpers/cache');
 const { MAX_CACHED_THREADS } = require('../constants');
```
Comment (Fahad-Mahmood) [inline controllers/dagora.controllers.js]: we are using constant ```MAX_CACHED_THREADS``` with redis only, this can be removed as well.

Comment (abrehmaaan) [reply controllers/dagora.controllers.js]: Okay

Comment (abrehmaaan) [reply controllers/dagora.controllers.js]: Removed

**Comment by abrehmaaan** [comment]
> looks good, but please delete the code and files instead of commenting.

Deleted

**Review by Fahad-Mahmood — APPROVED** [review]

**Pull request merged**